de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/json/AnyTypeAdapter.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.dvlm.json;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+/**
+ * This XmlAdapter can be used to marshal any generic Java Object. 
+ * This will result in a "type" attribute (XML) or key value pair (JSON)
+ * added to the marshalled object, which is needed for unmarshalling.
```

Comment:
This class probably needs some more description as to why we need it.
If I recall correctly it involved something with generics right?

Reply:
Yes I improved the comment, please look into it to check if it is clear now.